### PR TITLE
💄  make nav-bar

### PR DESCRIPTION
**Pull request by ashby-git**
💄 Combine nav-bar elements to make nav-bar

### CODE REVIEW

File: src/components/nav-bar/nav-bar.component.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import OnlineShopButton from './online-shop-button/online-shop-button.component.jsx';
+import './nav-bar.styles.scss';
+
+const NavBar = () => (
+
+    <div className="nav-bar-container">
+        <div className="items">
+            <NavBarItem>Concept</NavBarItem>
+        </div>
+        <div className="items">
```
Comment (DanArcherOnline) [inline src/components/nav-bar/nav-bar.component.jsx:12]: items is too much of a generic name. 
also remember its not an item itself, its just containing one for layout purposes.

File: src/components/nav-bar/nav-bar.component.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <NavBarItem>Quality</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>Products</NavBarItem>
+        </div>
+        <div className="logo-container">
+            <div className="logo" />
```
Comment (DanArcherOnline) [inline src/components/nav-bar/nav-bar.component.jsx:19]: possibly use an img

File: src/components/nav-bar/nav-bar.component.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <div className="items">
+            <NavBarItem>Shops</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>News</NavBarItem>
+        </div>
+        <div className="items">
+            <NavBarItem>Contact</NavBarItem>
+        </div>
+        <div className="shop-button">
```
Comment (DanArcherOnline) [inline src/components/nav-bar/nav-bar.component.jsx:30]: same as the above comment, its not a shop button itself.
choose a more descriptive name.

File: src/components/nav-bar/nav-bar.styles.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.nav-bar-container {
+    width: auto;
+    height: auto;
```
Comment (DanArcherOnline) [inline src/components/nav-bar/nav-bar.styles.scss:3]: just make sure these properties are necessary. if something changes, plop them back in.

File: src/components/nav-bar/nav-bar.styles.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 8vw;
+}
+
+.logo {
+    background: url(../../assets/logo.svg);
+    width: 112px;
+    height: 112px;
+}
```
Comment (DanArcherOnline) [inline src/components/nav-bar/nav-bar.styles.scss:14]: this wont be necessary if you use an img tag

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]

File: src/components/nav-bar/nav-bar.component.jsx (hunk 1 above)
Comment (ashby-git) [reply src/components/nav-bar/nav-bar.component.jsx]: changed to nav-bar-item-container

File: src/components/nav-bar/nav-bar.component.jsx (hunk 2 above)
Comment (ashby-git) [reply src/components/nav-bar/nav-bar.component.jsx]: changed to img tag

File: src/components/nav-bar/nav-bar.component.jsx (hunk 3 above)
Comment (ashby-git) [reply src/components/nav-bar/nav-bar.component.jsx]: changed to shop-buton-container

File: src/components/nav-bar/nav-bar.styles.scss (hunk 4 above)
Comment (ashby-git) [reply src/components/nav-bar/nav-bar.styles.scss]: after testing, it was irrelevant, so its been deleted

File: src/components/nav-bar/nav-bar.styles.scss (hunk 5 above)
Comment (ashby-git) [reply src/components/nav-bar/nav-bar.styles.scss]: deleted

**Review by DanArcherOnline — APPROVED** [review]
lgtm!

**Pull request merged**